tools/p/README.md

```diff
@@ -0,0 +1,5 @@
+Usage
```

Comment:
nit: maybe a short description that indicates that `p` is a containerized drop-in replacement for the p program?

Reply:
fixed